schemas/uptime-configs.v1.schema.json

```diff
@@ -0,0 +1,37 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "uptime_config",
+  "$ref": "#/definitions/CheckConfig",
+  "definitions": {
+    "CheckInterval": {
+      "title": "check_interval",
+      "description": "The interval between each check run in seconds.",
+      "type": "number",
+      "enum": [60, 300, 600, 1200, 1800, 3600]
+    },
+    "CheckConfig": {
+      "title": "check_config",
+      "description": "A message containing the configuration for a check to scheduled and executed by the uptime-checker.",
+      "type": "object",
+      "additionalProperties": true,
+      "properties": {
+        "subscription_id": {
+          "description": "UUID of the subscription that this check config represents.",
+          "type": "string"
+        },
+        "interval": {
+          "$ref": "#/definitions/CheckInterval"
+        },
+        "timeout": {
+          "description": "The total time we will allow to make the request in seconds.",
+          "type": "number"
+        },
```

Comment:
If we go over the max will we just clamp it on the uptime checker side? Or I guess maybe we should just enforce it in sentry instead...

Reply:
👍